packages/collaboration/src/userinfopanel.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { ReactWidget } from '@jupyterlab/apputils';
+
+import * as React from 'react';
+
+import { ICurrentUser, IUser } from './tokens';
+import { UserIconComponent } from './components';
+import { Panel } from '@lumino/widgets';
+
+export class UserInfoPanel extends Panel {
+  private _profile: ICurrentUser;
+  private _body: UserInfoBody;
+
+  constructor(user: ICurrentUser) {
+    super({});
+    this.title.label = 'User info';
+    this.title.caption = 'User information';
```

Comment:
This requires translation - I would advice to do it in the plugin activation to avoid bringing down the translator object.